docs/annotations.md

```diff
@@ -73,3 +77,90 @@ spec:
           serviceName: websocket-repeater
           servicePort: 80
 ```
+
+# Connection Draining
+`connection-draining`: This annotations allows to specify whether to enable connecitn draining and the timeout after which
```

Comment:
s/annotations/annotation

A few typos:
`connection-draining`: This annotation allows a user to specify connection draining and timeout ....